aws-athena/views/qc-vw_iasworld_sales_price_diff_sale_mydec.sql

```diff
@@ -3,6 +3,7 @@
 SELECT
     REGEXP_REPLACE(iasworld.instruno, '[^0-9]', '') AS document_number,
     SUBSTR(iasworld.saledt, 1, 4) AS taxyr,
+    iasworld.parid,
```

Comment:
Neglecting to select this column was resulting in output for this test that was missing a parid, as well as township (since that's queried off of parid and taxyr). Not directly related to this PR, but I went ahead and fixed it here anyway.